<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>1065</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Random_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>0</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заповнити масив рандомно</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кількість елементів</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ssssss">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>104</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="farray">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>610</y>
      <width>1061</width>
      <height>0</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="legendComboBox">
    <property name="geometry">
     <rect>
      <x>531</x>
      <y>1099</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="animatedComboBox">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>1099</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="themeComboBox">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>1099</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="antialiasCheckBox">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>1105</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Anti-aliasing</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>325</width>
      <height>516</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="a_bubble">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортування Бульбашкою</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="time_b">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="a_selectionsort">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортування Вибіркою</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="time_selection">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="a_shella">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортування Шелла</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="time_shella">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="a_qssort">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>QS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="time_qt">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="time_counting">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="a_cs">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Counting Sort</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="time_merge">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="a_merge">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
